broker/config/database.js

```diff
@@ -1,18 +1,17 @@
 // Read private configuration settings from .env file into process.env
 //
-const dotenv = require("dotenv");
-dotenv.config();
+require("dotenv").config({ path: `${__dirname}/../../.env` });
 
 // Export configuration settings with some values set to defaults.
 // NOTE: login/password settings should not be defaulted.
 //
 module.exports = {
     // For Sequelize (defaults below for PostgreSQL)
     //
-    dbName: process.env.DB_NAME || 'massvaxx',
-    dialect: process.env.DB_DIALECT || 'postgres',
-    host: process.env.DB_HOST || 'localhost',
-    password: process.env.DB_PASSWORD,
-    port: process.env.DB_PORT || 5432,
-    username: process.env.DB_USERNAME
+    dbName: process.env.BROKER_DB_NAME || 'massvaxx',
+    dialect: process.env.BROKER_DB_DIALECT || 'postgres',
+    host: process.env.BROKER_DB_HOST || 'localhost',
+    password: process.env.BROKER_DB_PASSWORD,
+    port: process.env.BROKER_DB_PORT || 6432,
```

Comment:
Please leave this port default set to 5432 (the default Postgres port)!

Reply:
I changed the default port to match the defaults needed for the third build method. Since both databases need to be exposed locally for the tests, I had to use a different port. Do you still want me to use the default Postgres port?